WTWSMS/events/religious_events.txt

```diff
@@ -783,7 +801,7 @@ character_event = {
 			trait = cynical
 		}
 		modifier = {
-			factor = 0.25
+			factor = 0.1
```

Comment:
What you want to do is making this event less likely to fire,. I disagree on doing that.